deploy/crds/operators.ibm.com_v1alpha1_switcheritem_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operators.ibm.com/v1alpha1
+kind: SwitcherItem
+metadata:
+  name: example-switcheritem
```

Comment:
this should be the default switcher item for common web ui. It should define the switcher item for us and should only have the dashboard url to show up.